pkg/virt-api/webhooks/validating-webhook/validating-webhook_test.go

```diff
@@ -1140,6 +1140,83 @@ var _ = Describe("Validating Webhook", func() {
 			Expect(causes[0].Field).To(Equal("fake.domain.devices.disks[0].name"))
 			Expect(causes[1].Field).To(Equal("fake.domain.devices.disks[0]"))
 		})
+		It("should accept correct tablet input device", func() {
```

Comment:
You can also create a table-based test here and and pass in once `virtio` and once `usb`. If you don't want to create a table, then make the name of the two tests non-overlapping and more specific. E.g. `with usb bus` and `with virtio bus`.

Reply:
In general the tests here sem to be a good cantidate for a table based test.